modules/rosa-edit-objects.adoc

```diff
@@ -85,11 +85,32 @@ $ rosa edit ingress --cluster=<cluster_name> | <cluster_id> [arguments]
 |--cluster
 |Required: The name or ID (string) of the cluster to which the ingress will be added.
 
+|--cluster-routes-hostname  
+|Components route hostname for OAuth, console, download.
+
+|--cluster-routes-tls-secret-ref 
+|Components route TLS secret reference for OAuth, console, download.
+
+|--excluded-namespaces 
+|Excluded namespaces for ingress. Format is a comma-separated list `value1, value2...`. If no values are specified, all namespaces will be exposed.
+
 |--label-match
 |The label match (string) for ingress. The format must be a comma-delimited list of key=value pairs. If no label is specified, all routes are exposed on both routers.
 
+|--lb-type  
+|Type of Load Balancer. Options are `classic`, `nlb`.
+
+|--namespace-ownership-policy
+|Namespace Ownership Policy for ingress. Options are `Strict`, `InterNamespaceAllowed`. Default is `Strict`.
```

Comment:
Also here to separate the options since there aren't 3 shouldn't the comma be replaced with "and"?

```suggestion
|Namespace Ownership Policy for ingress. Options are `Strict` and `InterNamespaceAllowed`. Default is `Strict`.
```